scripts/ci_run_test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -x
+
+TEST_RESULT_FILE="/root/project/test_results.xml"
+
+# copy unity license file
+mkdir /root/.local/share/unity3d/Unity
+cp /root/project/Unity_license.ulf /root/.local/share/unity3d/Unity/Unity_lic.ulf
```

Comment:
Why is unity license needed?

Reply:
you need to register and create at least personal user for running Unity on this Docker, it actually seems different from what happens in dev machine, but maybe the UI is creating that for you or something. anyway it's not working without this, I've use instructions here:
https://github.com/GabLeRoux/unity3d-ci-example
https://gitlab.com/gableroux/unity3d